src/server/index.jsx

```diff
@@ -303,11 +304,14 @@ server
         // Set derivedPageType based on matched route
         derivedPageType = pageType || derivedPageType;
 
+        const remoteConfig = await getRemoteConfig(service);
+
         const data = await getInitialData({
           path: url,
           service,
           variant,
           pageType,
+          remoteConfig,
```

Comment:
Do we just want to pass this into `getInitialData`, or pass it into the app at the top level so every page type has access to toggles without needing to update `getInitialData`? I understand that some `getInitialData` functions will need to use remoteConfig as well before we render the react app.